aqua/kitty/Portfile

```diff
@@ -6,7 +6,7 @@ PortGroup           python 1.0
 
 github.setup        kovidgoyal kitty 0.20.1 v
 github.tarball_from releases
-revision            2
+revision            3
```

Comment:
If you're only fixing a build failure, then you should not increase the revision.